src/canvas/SelectableCanvas.ts

```diff
@@ -1052,11 +1051,8 @@ export class SelectableCanvas<
     // until we call this function specifically to search inside the activeGroup
     while (i--) {
       const objToCheck = objects[i];
-      const pointerToUse = objToCheck.group
-        ? this._normalizePointer(objToCheck.group, pointer)
-        : pointer;
-      if (this._checkTarget(pointerToUse, objToCheck, pointer)) {
-        target = objects[i];
+      if (this._checkTarget(objToCheck, pointer)) {
+        target = objToCheck;
```

Comment:
ok this is cool, i wanted to do it since long.

Reply:
1 over 32 files i have nothing to say

Reply:
> 1 over 32 files i have nothing to say

Don't understand